packages/table/src/table-head-cell.css

```diff
@@ -15,4 +15,5 @@ governing permissions and limitations under the License.
 :host {
     display: block;
     flex: 1;
+    block-size: auto;
```

Comment:
can't this come from css?

Reply:
It's needed in this case because the display is changed to `flex`, which changes the behavior so it no longer works like a min-height like it does with `display: table-cell`.